stage0/run.go

```diff
@@ -379,6 +380,26 @@ func preparedWithPrivateUsers(dir string) (string, error) {
 	return string(bytes), nil
 }
 
+func addResolvConf(cfg RunConfig, rootfs string) {
+	if len(cfg.DNS) == 0 {
```

Comment:
IMHO it'd be better to read if this condition would also or even only be used before the invocation of `addResolvConf(..)`